### PR TITLE
Created new styleguide for angular 1.6

### PR DESCRIPTION
**Pull request by mravinale**
## Background
Created new Guide based on Todd Motto style guide https://github.com/toddmotto/angularjs-styleguide

## Changes done
 n/a

## Pending to be done
n/a

## Notes
n/a

## Required Approval
- @AlphaGit @kilisoria @nmartello-makingsense

### CODE REVIEW

**Comment by AlphaGit** [comment]
I will read it -- so just give me a day or two. I agree with @matiasbeckerle -- I don't want to challenge every single point, just want to make sure that it doesn't directly contradict anything we have in the rest of the guidelines.

Aside from that, I also agree that the link in the Readme would be great to have in this PR.

**Comment by kilisoria** [comment]
Hi guys! sound good! Me too. Let me some days to review it.

File: Angular(1.6)/README.md [hunk 1]
```diff
@@ -0,0 +1,772 @@
...
+import templateUrl from './todo.html';
+
+export const TodoComponent = {
+  bindings: {
+    todoData: '<'
+  },
+  templateUrl,
+  controller: class TodoComponent {
+    constructor() {
+      'ngInject'; // Not actually needed but best practice to keep here incase dependencies needed in the future
```
Comment (AlphaGit) [inline Angular(1.6)/README.md]: Minor typo: should be "in case".

**Comment by AlphaGit** [comment]
@kilisoria Have you had a chance to take a look?

**Comment by kilisoria** [comment]
yes @AlphaGit , I had put a comment a time ago hmm anyway I had look at this is OK for me. This repository is more up to date and good reference for the topic.

**Comment by AlphaGit** [comment]
@kilisoria Thanks a lot!

**Pull request merged**